src/server/pkg/storage/fileset/option.go

```diff
@@ -54,8 +53,9 @@ func WithMaxOpenFileSets(max int) StorageOption {
 // UnorderedWriterOption configures an UnorderedWriter.
 type UnorderedWriterOption func(f *UnorderedWriter)
 
-// WithRenewer sets the UnorderedWriter's Renewer.
-func WithRenewer(ttl time.Duration, r *Renewer) UnorderedWriterOption {
+// WithRenewal configures the UnorderedWriter to renew subfileset paths
+// with the provided renewer.
+func WithRenewal(ttl time.Duration, r *Renewer) UnorderedWriterOption {
```

Comment:
😐

Reply:
Yeah, I had a bit of a name conflict. I think this is a reasonable change.